arquillian/api/src/main/java/org/wildfly/swarm/arquillian/adapter/UberjarSimpleContainer.java

```diff
@@ -130,7 +130,15 @@ public void start(Archive<?> archive) throws Exception {
                 .fractionDetectionMode(BuildTool.FractionDetectionMode.never)
                 .bundleDependencies(false);
 
-        final String additionalModules = System.getProperty(SwarmInternalProperties.BUILD_MODULES);
+        String additionalModules = System.getProperty(SwarmInternalProperties.BUILD_MODULES);
+
+        // See https://issues.jboss.org/browse/SWARM-571
+        if(null==additionalModules) {
+            // see if we can find it
+            File modulesDir = new File("target/classes/modules");
+            additionalModules = modulesDir.exists() ? modulesDir.getAbsolutePath() : null;
+        }
+
```

Comment:
Currently if the system property is set, nothing from target/classes/modules would be included.

Maybe there should be multiple additionalModule dirs added?